source/class/qx/tool/cli/commands/Deploy.js

```diff
@@ -0,0 +1,158 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2020 Zenesis Ltd https://www.zenesis.com
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+     * John Spackman (john.spackman@zenesis.com, @johnspackman)
+
+************************************************************************ */
+require("./Compile");
+
+require("@qooxdoo/framework");
+const fs = require("fs");
+const path = require("upath");
+const stream = require("stream");
+const util = require("util");
+const pipeline = util.promisify(stream.pipeline);
+
+/**
+ * Build and deploy a project
+ * 
+ */
+qx.Class.define("qx.tool.cli.commands.Deploy", {
+  extend: qx.tool.cli.commands.Command,
+  statics: {
+    getYargsCommand: function() {
+      return {
+        command: "deploy [options]",
+        describe: "deploys qooxdoo application(s)",
+        builder: {
+          "out": {
+            describe: "Output directory for the deployment",
+            demandOption: true
+          },
+          "app-name": {
+            describe: "The name of the application to deploy (default is all apps)",
+            nargs: 1,
+            type: "string"
```

Comment:
Maybe a comma-separated list of app names would make sense. But one can also run `qx deploy --app-name={app1,app2}` in a shell.